tpm2/test/tpm2_test.go

```diff
@@ -1511,6 +1511,43 @@ func TestPlainImport(t *testing.T) {
 	}
 }
 
+func TestClear(t *testing.T) {
```

Comment:
IIRC, this test will mess up a machine if it is run inadvertently?

If so, can you add a new flag and use that to hide running this test?

Reply:
That's a good point - added flag which will skip test if not set